OmniKit/PumpManager/PodComms.swift

```diff
@@ -41,52 +49,147 @@ class PodComms: CustomDebugStringConvertible {
         self.messageLogger = nil
     }
     
-    private func assignAddress(address: UInt32, commandSession: CommandSession) throws -> PodState {
+    // Handles all the common work to send and verify the version response for the two pairing commands, AssignAddress and SetupPod
```

Comment:
Could you document which errors get thrown from this?

Reply:
Thanks for taking care of his for me Pete!